ndc-lambda-sdk/test/inference/basic-inference/basic-inference.test.ts

```diff
@@ -311,13 +311,40 @@ describe("basic inference", function() {
                   name: "IThing"
                 }
               },
+              {
+                argumentName: "aliasedInterface",
+                description: null,
+                type: {
+                  type: "named",
+                  kind: "object",
+                  name: "IThing"
+                }
+              },
               {
                 argumentName: "genericInterface",
                 description: null,
                 type: {
                   type: "named",
                   kind: "object",
-                  name: "IGenericThing"
+                  name: "IGenericThing<string>"
```

Comment:
What does this pop out as in the schema?

Reply:
`IGenericThing<string>`. That is the the type name. The LSP should be escaping any GraphQL incompatible characters when it creates the types, etc. The name in NDC can be anything, there's no restrictions.